index.html

```diff
@@ -22,14 +22,16 @@
   <body>
     <header>
       <nav id="mobile-nav">
-        <a href="" id="logo">Welcome</a>
+        <a href="#" id="logo">Welcome</a>
         <i class="fa-solid fa-bars" id="hamburger"></i>
       </nav>
-      <nav id="desktop-nav">
+      <nav id="desktop-nav" class="desktop-nav">
+        <div id="icon"
+        ><i class="fa-solid fa-xmark"></i></div>
         <ul id="nav-links">
           <li><a href="index.html">Portfolio</a></li>
-          <li><a href="#about-section">About</a></li>
-          <li><a href="#contact-sections">Contact</a></li>
+          <li><a href="#about-section" id="about">About</a></li>
+          <li><a href="#contact-sections" id="contact">Contact</a></li>
         </ul>
       </nav>
```

Comment:
Nice job with the `menu bar` 👍. Please stick to the design provided for the `menu bar` option popup window. Make sure to remove the underline of option text and increase the font size.

| Your Design | Figma Design |
|-------------|---------------|
| <img width="310" alt="image" src="https://user-images.githubusercontent.com/34578631/173570520-6064cb8c-d27f-4f52-945a-d39024ee9fcd.png"> | <img width="338" alt="image" src="https://user-images.githubusercontent.com/34578631/173570557-f476e93a-44ba-47cd-9f47-dd7b96127eca.png"> |